src/psyclone/psyir/backend/visitor.py

```diff
@@ -154,21 +169,27 @@ def _visit(self, node):
         :param node: A PSyIR node.
         :type node: :py:class:`psyclone.psyir.nodes.Node`
 
+        :returns: representation of the sub-tree with `node` at its root.
```

Comment:
Maybe "text representation of the given PSyIR node sub-tree"

Reply:
Done.